config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  get 'home/index'
```

Comment:
this is unnecessary because the `root` path points here. also, I'd recommend moving `root` to the top as routes are matched from top-down.

Reply:
I noticed that but wasn't certain if the 2 syntax styles provided the exact same functionality. The line was added when I generated the controller with 'index' from the command line.